salt/modules/pillar.py

```diff
@@ -62,6 +63,24 @@ def get(key,
             then merging will be skipped and the retrieved value will be
             returned. Earlier releases raised an error in these cases.
 
+    deep_merge_lists : ``None``
```

Comment:
Greh, GitHub isn't refreshing automatically to notify me of new comments. 

> Well, what we could do to make this more intuitive is we could make the merge argument remove duplicates when default is a list. The code for that would be right here in pillar.get.

That is exactly what the [current code](https://github.com/saltstack/salt/blob/develop/salt/modules/pillar.py#L131-L146) already does when both `default` and the returned value are lists. Which is why to me it would be confusing just to call this `merge_lists`. That's _already_ done by `merge`.